docs/data_onboarding/antivirus_antimalware/sophos_endpoint_protection.md

```diff
@@ -54,6 +54,48 @@ Cyences version 1.6 utilizes Sophos Central API to collect information about Sop
 8. Refer to the Sophos Central documentation link below for further assistance.  
     *[https://developer.sophos.com/getting-started](https://developer.sophos.com/getting-started) 
 
+## Sophos Partner Portal API Configuration For A Specific Client
+
+1. Login to sophos Partner Portal.
+
+2. Click on **Manage Groups**.
+
+3. Click on **Firewalls**.
```

Comment:
I don't think you need to go to "Firewalls" section.

Reply:
Without clicking on the firewalls tab on sophos partner portal you won't be able to check the list of clients on the portal, but for sophos central no need its only required for sophos partner portal to be able to review all customer and select the customer desire to generate the API for this customer